charts/ingress-nginx/templates/controller-psp.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if (semverCompare "<1.25.0-0" .Capabilities.KubeVersion.Version) }}
-{{- if and .Values.podSecurityPolicy.enabled (empty .Values.controller.existingPsp) -}}
+{{- if and .Values.podSecurityPolicy.enabled (not .Values.global.podSecurityStandards.enforced) (empty .Values.controller.existingPsp) -}}
```

Comment:
As Giant Swarm is dropping PSP in recent releases and upstream is planning on doing so, that diff will probably disappear in the near future.